config.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  jnlp-port:
+    type: int
+    default: 50000
+    description: TCP port on which to listen for incoming connections from agents.
```

Comment:
What's the reasoning behind ? We should add any config param consciously.

Reply:
I thought I had to maintain backwards compatibility, it was discussed that it doesn't need to be backwards compatible :), this will be hardcoded in the j2 template!